<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComputerDialogDesktop</class>
 <widget class="QWidget" name="ComputerDialogDesktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>251</width>
    <height>189</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Codec</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_codec"/>
       </item>
       <item>
        <widget class="QLabel" name="label_color_depth">
         <property name="text">
          <string>Color depth:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="combo_color_depth"/>
       </item>
       <item>
        <widget class="QLabel" name="label_compression_ratio">
         <property name="text">
          <string>Compression ratio:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_fast">
           <property name="text">
            <string>Fast</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="slider_compression_ratio">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>22</number>
           </property>
           <property name="value">
            <number>8</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksAbove</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_best">
           <property name="text">
            <string>Best</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>17</width>
           <height>15</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QCheckBox" name="checkbox_block_remote_input">
         <property name="text">
          <string>Block remote input</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_cursor_shape">
         <property name="text">
          <string>Enable cursor shape</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_clipboard">
         <property name="text">
          <string>Enable clipboard</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_desktop_effects">
         <property name="text">
          <string>Disable desktop effects</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_desktop_wallpaper">
         <property name="text">
          <string>Disable desktop wallpaper</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkbox_font_smoothing">
         <property name="text">
          <string>Disable font smoothing</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>105</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
